### ISSUE TITLE
Tiny-Imagenet-200 accuracy is 0.50 

### ISSUE THREAD

**Issue by khawar-islam** [snu-mllab/PuzzleMix#12]
Dear @Janghyun1230 

Based on your training script for tiny-ImageNet-200, I have trained a model from scratch but the accuracy is completely stuck at 0.50 from the first epoch to the last epoch. I did not change anything in your architecture and given the script. **Do you have any idea why it's happening?**

**Training Script**
`python main.py --dataset tiny-imagenet-200 --data_dir data/tiny-imagenet-200 --root_dir [save_path] --labels_per_class 500 --arch preactresnet18  --learning_rate 0.2 --momentum 0.9 --decay 0.0001 --epochs 600 --schedule 300 450 --gammas 0.1 0.1 --train mixup --mixup_alpha 1.0 --graph True --n_labels 3 --eta 0.2 --beta 1.2 --gamma 0.5 --neigh_size 4 --transport True --t_eps 0.8 --clean_lam 1`

**Log file**

[log.txt](https://github.com/snu-mllab/PuzzleMix/files/10456893/log.txt)

**Comment by khawar-islam** [comment]
@Janghyun1230 Would it be possible for you to check my question?

**Comment by Janghyun1230** [comment]
Hello islam, sorry for late response.
One of my suspects is the dataset label issue (due to version or preprocessing error). The 0.5 accuracy means the random guessing or evaluation with identical label (there are total 200 classes).

Anyway, my suggestion is to narrow down the problem.
- check that experiments on CIFAR have the same issue
- print and check dataset labels and batch labels span over 0~199
- If the labels are ok, then download the trained model and evaluate them (https://drive.google.com/drive/folders/1jxCib7eSoKBthNGyke7lQuVHgkl21mtZ?usp=sharing). 
- try vanilla training without mixup

Hope this helps. 
Jang-Hyun

**Comment by khawar-islam** [comment]
@Janghyun1230 thank you, problem solved
